python/sklearnserver/sklearnserver/model.py

```diff
@@ -46,8 +47,8 @@ def load(self) -> bool:
         self.ready = True
         return self.ready
 
-    def predict(self, request: Dict) -> Dict:
-        instances = request["instances"]
+    def predict(self, payload: Dict, headers: Dict[str, str] = None) -> Dict:
```

Comment:
changing this is fine, but just want to make sure it can still work without adding the headers to be backwards compatible.